src/app.tsx

```diff
@@ -184,7 +260,7 @@ export default function Chat() {
             <Toggle
               toggled={showDebug}
               aria-label="Toggle debug mode"
-              onClick={() => setShowDebug((prev) => !prev)}
+              onClick={() => setShowDebug(!showDebug)}
```

Comment:
i think the prev way of doing this is considered safer, did you change it for readability?